lib/api/v3/work_packages/work_package_representer.rb

```diff
@@ -251,7 +251,7 @@ def initialize(model, current_user:, embed_links: false)
 
         linked_property :parent,
                         path: :work_package,
-                        title_getter: -> (*) { represented.parent.subject },
+                        title_getter: -> (*) { represented.parent.to_s },
```

Comment:
This strikes me as inconsistent as we do not present work packages or other resources this way anywhere else. Therefore I'd rather stick with the subject. If that means that we need to do extra work, in order to display the parent the way that we do right now, then again, let's first try to clarify how it is supposed to be presented.